spec/poros/place_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe Place do
```

Comment:
In the cleaner branch I set up (deployment), I created a fixture file with this data and just did a File.read. So, once we switch main over this will end up looking different so if you had a similar thought, no need to reinvent the wheel!